tests/ci_build/deploy/aws

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
This file has no type (.py)

Reply:
This is a copy of the driver script for awscli, as the installation in user space doesn't include it. It's usually at `/usr/local/bin/aws` but slaves don't have write access to this path.